test/Test/Cardano/Prelude/Helpers.hs

```diff
@@ -24,6 +26,17 @@ assertEitherIsRight func val = case func val of
   Left  err -> failWith Nothing (show $ sformat build err)
   Right _   -> success
 
+assertMaybeIsJust :: MonadTest m => (a -> Maybe b) -> a -> m ()
+assertMaybeIsJust func val = case func val of
+  Nothing -> failWith Nothing "Nothing"
```

Comment:
I would put a `HasCallStack` constraint on each of these functions and also put `withFrozenCallStack` before `failWith ...`. `Hedgehog` will not show the failures explicitly otherwise. I was doing this for the other functions in the module on a separate branch but feel free to update the others similarly. :)